invenio_rdm_records/records/systemfields/tombstone.py

```diff
@@ -178,10 +206,16 @@ def __set__(self, record, value):
 
         self._set_cache(record, tombstone)
 
+    def __delete__(self, record):
+        """Remove tombstone."""
+        self.mark_to_be_removed(record)
```

Comment:
minor/question: shouldn't doing `record.tombstone = None` also call `self.mark_to_be_removed(record)`?

Reply:
yes it should, done